runtimes/protocol/identity-management.ts

```diff
@@ -131,7 +157,7 @@ export const updateProfileRequestType = new ProtocolRequestType<
 >('aws/identity/updateProfile')
 
 // getSsoToken
-export type SsoTokenId = string // Opaque identifier
+export type CredentialId = string // Opaque identifier
```

Comment:
This is an exported type, could this cause issues with existing clients?

Reply:
They are distinct types (even though they are both currently just aliases of string) and should be kept separate.